### PR TITLE
add trait trashcanable

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-libs/pull/15
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1026
https://github.com/open-orchestra/open-orchestra-model-interface/pull/108
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/411

### CODE REVIEW

File: MongoTrait/TrashCanable.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace OpenOrchestra\MongoTrait;
+
+/**
+ * Class TrashCanable
+ */
+trait TrashCanable
```
Comment (ngilain) [inline MongoTrait/TrashCanable.php]: Maybe deletable ? or removable ?
TrashCanable hurts me a lot :p

Moreover TrashCan is just a way to display the deleted docs, which are deleted, with or without the trashcan

Comment (nicolasThal) [inline MongoTrait/TrashCanable.php]: SoftDeleteable ?

It will be more close to the name from the doctrine behaviour

**Comment by alavieille** [comment]
@itkg-ngilain

**Comment by nicolasThal** [comment]
:+1:

**Comment by ngilain** [comment]
:+1:

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**